tests/components/rfxtrx/test_init.py

```diff
@@ -48,7 +49,9 @@ def test_default_config(self):
             {"sensor": {"platform": "rfxtrx", "automatic_add": True, "devices": {}}},
         )
 
-        assert len(rfxtrx.RFXOBJECT.sensors()) == 2
+        time.sleep(1)  # Dummy startup is slow
+
+        assert len(self.hass.data[rfxtrx.DATA_RFXOBJECT].sensors()) == 2
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)